src/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useNavigate } from "react-router";
+
+import Button from "../Button";
+
+import "./courseCard.scss";
+
+const CourseCard = (props) => {
```

Comment:
use destructurization
it will avoid repeated action as props.someProps
const {id, name, ....} = props